client/lib/plans/features-list.js

```diff
@@ -1083,7 +1083,7 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_ANTISPAM_V2 ]: {
 		getSlug: () => FEATURE_ANTISPAM_V2,
-		getTitle: () => i18n.translate( 'Comment and form protection' ),
+		getTitle: () => i18n.translate( 'Comment and form spam protection' ),
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.